controllers/managedfusionoffering_controller.go

```diff
@@ -45,11 +47,14 @@ const (
 	subscriptionName  = "managed-fusion-offering"
 )
 
+var errRestart = errors.New("Restart")
```

Comment:
This is not what I was requesting. 
Returning an error for a restart is exactly the same as just `os.Exist` in-place.
The point was to create a new class of errors (preferably in a util file) that forces the operator to exist. 
This does not mean that the error does not have a message or a reason, or even a nested error!